crawlers/mooncrawl/mooncrawl/contract/cli.py

```diff
@@ -117,7 +119,7 @@ def run_crawler_desc(
 
 def handle_parser(args: argparse.Namespace):
     with yield_db_session_ctx() as session:
-        w3 = connect(AvailableBlockchainType.ETHEREUM)
+        w3 = connect(AvailableBlockchainType.ETHEREUM, access_id=args.access_id)
```

Comment:
@kompotkot : Does this work even if we want to connect directly to Ethereum node (skipping node balancer)?

Reply:
Yes, if None provided by default it do not change behaviour.